.gitignore

```diff
@@ -3,3 +3,6 @@
 /example
 /vendor
 composer.lock
+
+.idea
```

Comment:
I recommend adding any IDE related files to **global gitignore**.

Here is how: https://stackoverflow.com/a/7335487/1348344

Reply:
Could remove this line please? It could motivate people to add all IDE's files instead of ignoring them globally, which would be mess.